server/v2/stf/helper.go

```diff
@@ -0,0 +1,31 @@
+package stf
+
+import (
+	"context"
+
+	"cosmossdk.io/core/store"
+	"cosmossdk.io/server/v2/stf/mock"
+)
+
+// There some field not be exported
```

Comment:
I personally do not like that we need to expose this.
If we make this public, anyone can use and abuse it.
Before it was in a `export_test.go` file, so it is public but only for stf tests, which is okay.

cc @testinginprod for thoughts.

Reply:
this is no bueno

Reply:
Removed stf exports, assert by store height & commit info. Ty guys